docs/querying/sql-functions.md

```diff
@@ -1154,11 +1154,29 @@ Returns the following:
 
 ## GREATEST
 
-`GREATEST([expr1, ...])`
+Returns the maximum value from the provided expressions. The expressions must all be convertible to a common data type.
 
-**Function type:** [Scalar, reduction](sql-scalar.md#reduction-functions)
+* **Syntax:** `GREATEST([expr1, ...])`
+* **Function type:** Scalar, reduction
+
+<details><summary>Example</summary>
+
+The following example returns the greatest value between the numeric constant `PI`, the integer number `4` and the double `-5.0`, which are all interpreted as doubles. 
```

Comment:
```suggestion
The following example returns the greatest value between the numeric constant `PI`, the integer number `4`, and the double `-5.0`, which are all interpreted as doubles. 
```
This sentence is too long. I suggest that you split it into two. Also, "interpreted as doubles" by whom? Clarify and, if possible, use active voice.